db/migrate/20250925143706_add_eligibility_lists.rb

```diff
@@ -0,0 +1,14 @@
+class AddEligibilityLists < ActiveRecord::Migration[7.2]
+  disable_ddl_transaction!
```

Comment:
Since this is adding an empty table, I think we can adding the index in a blocking fashion (since it will be nanoseconds) which means we don't need to disable DDL transactions